client/components/vertical-nav/index.jsx

```diff
@@ -2,6 +2,12 @@
  * External dependencies
  */
 import React from 'react';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
```

Comment:
I think we shouldn't merge changes in this file. Can you split this to its own file so we could merge only code that  enabled stylizer?